src/Microsoft.DotNet.ApiSymbolExtensions/Filtering/CompositeSymbolFilter.cs

```diff
@@ -15,6 +15,8 @@ public class CompositeSymbolFilter : ISymbolFilter
     {
         private readonly List<ISymbolFilter> _innerFilters = new();
 
+        public List<ISymbolFilter> Filters { get => _innerFilters; }
```

Comment:
Same as in AccessibilitySymbolFilter, remove the backing field and let the compiler generate it automatically:

```suggestion
        public List<ISymbolFilter> Filters { get; }
```